components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/network/NetworkSettingsConfigurator.java

```diff
@@ -54,4 +54,10 @@ private static Properties loadProperties(Path propertiesFile) throws IOException
             return properties;
         }
     }
+
+    public static class FailedToLoadNetworkSettingsException extends RuntimeException {
```

Comment:
This exception type can be removed.

Reply:
Removed in 359c6393661f7df0d2be74751d5d3ec6d44bc14b